decidim-bulletin-board-client-ruby/decidim-bulletin_board.gemspec

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative 'lib/decidim/bulletin_board/version'
+
+Gem::Specification.new do |s|
+  s.name          = 'decidim-bulletin_board'
```

Comment:
woudn't it be better to name it `decidim-bulletin_board_client` to avoid confusion with the real bulletin board application?
This will imply at least renaming this file and the directories in `lib/decidim/bulletin_board_client`